RecoEgamma/EgammaIsolationAlgos/interface/EgammaL1TkIsolation.h

```diff
@@ -0,0 +1,64 @@
+#ifndef RECOEGAMMA_EGAMMAISOLATIONALGOS_EGAMMAL1TKISOLATION_H
```

Comment:
by convention, we don't use allcaps for this, but rather the same capitalization as in the filenames.

Reply:
done